src/components/Template/templateChangelogData.ts

```diff
@@ -9,6 +9,13 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "Prerelease",
+    version: "Prerelease",
+    type: "Update",
+    affects: ["Functionality", "Styles"],
+    notes: ["Major update to component and styles based on the Template TAD."],
```

Comment:
It's ok to be more descriptive here! Maybe not _all_ the details but definitely the major ones.